app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Realm/RealmStory.java

```diff
@@ -12,95 +12,133 @@
  * @see RealmObject
  */
 public class RealmStory extends RealmObject implements Story{
-    private RealmPublication publication;
-    private String publicationDate;
-    private String lastUpdated;
-    private RealmList<RealmAuthor> author;
-    private RealmReactions reactions;
-    private String fullText;
-    private String blurb;
+    private String datePublished;
+    private String brief;
+    private String headerImage;
+    private String publication;
+    private String dateEdited;
+    private int articleId;
     private String title;
-    private String thumbnailUrl;
+    private String content;
+    private RealmList authors;
 
 
     /* Default constructor for Realm */
     public RealmStory(){}
 
-    public RealmStory(RealmPublication publication, String publicationDate, String lastUpdated,
-                      RealmList<RealmAuthor> author, RealmReactions reactions, String fullText,
-                      String blurb, String title, String thumbnailUrl) {
+    public RealmStory(String datePublished, String brief, String headerImage, String publication,
```

Comment:
The method should be formatted properly. Also, we should definitely look into *not* having such a long constructor.